testsuite/oauth/authorize/base.py

```diff
@@ -1,10 +1,13 @@
 """ Authorize the SMART API.
 """
+import time
 from abc import ABCMeta
 from urllib import parse
+from pyvirtualdisplay import Display
```

Comment:
Fair enough -- I will follow all of your nitpicks if you update the lints to fail me for breaking them :-)